electrum/gui/kivy/uix/dialogs/cpfp_dialog.py

```diff
@@ -141,21 +134,20 @@ def update_text(self):
         self.ids.fee_target.text = target
         fee_per_kb = self.config.fee_per_kb()
         self.fee = self.get_child_fee_from_total_feerate(fee_per_kb)
-        if fee_per_kb is None:
+        if self.fee is None:
             self.ids.fee_for_child.value = "unknown"
         else:
             comb_fee = self.fee + self.parent_fee
-            comb_feerate = comb_fee / self.total_size 
-            self.ids.fee_for_child.value = f'{self.fee:.1f} sat'
-            self.ids.output_amount.value = f'{self.max_fee-self.fee:.1f} sat'
-            self.ids.total_fee.value = f'{self.fee+self.parent_fee:.1f} sat'
-            self.ids.total_feerate.value = f'{comb_feerate:.1f} sat/B'
+            comb_feerate = 1000 * comb_fee / self.total_size
+            self.ids.fee_for_child.value = self.app.format_amount_and_units(self.fee)
+            self.ids.output_amount.value = self.app.format_amount_and_units(self.max_fee-self.fee)
```

Comment:
One difference between the Qt GUI and here is, that in Qt if the user sets the fee to too high (over max_fee) the displayed text for "Output amount" is "", which I think is better than e.g. "0 BTC" which gets shown here. (In either case the user gets an error that the fee is too high if they try to proceed)
While neither is particularly good UX, I find it somewhat better to do what Qt does and set "Output amount" to "", so let's do that.

See pics:

<details>
 <summary>pic1</summary>

![t1](https://user-images.githubusercontent.com/29142493/132921673-efb0b9a9-f05c-48fd-8614-e2d209a1a19d.PNG)
</details>
<details>
 <summary>pic2</summary>

![t2](https://user-images.githubusercontent.com/29142493/132921682-108868fc-862f-4796-adf1-ed5ed598021a.PNG)
</details>

https://github.com/spesmilo/electrum/blob/bf6400ae30453306c4e8889647d92ddd2bc1d090/electrum/gui/qt/main_window.py#L3334